### ISSUE TITLE
Bird tweeting sounds are off

### ISSUE THREAD

**Issue by Richard-L** [Return-To-The-Roots/s25client#1246]
[bird-tweet-sfx.zip](https://github.com/Return-To-The-Roots/s25client/files/4726030/bird-tweet-sfx.zip)

^Compare these bird tweet sounds of original S2 and RttR. I've always found RttR's very weird sounding, like stuff I've never ever heard in the original S2.

Is it possible to fix this?

**Comment by Flamefire** [comment]
@Spikeone @Flow86 This might be due to wrong resampling. The `sound.scs` defines how it is resampled. How was this file created and could it be fixed (if that's the cause)?

**Comment by Flow86** [comment]
it was kind of manually, because some sounds didnt have the default 9600hz I think. but the file could be wrong.

**Comment by Flamefire** [comment]
> what about that fileformat for .bob overriding?

Sorry, I don't understand the connection to bob files.

**Comment by Flow86** [comment]
never mind, I thought about the format of the scs file for the bob linking stuff?

**Comment by Flamefire** [comment]
Not really because we have 1 line per target index which is unfeasible for the few thousand entries in the bob files when you only want to add/change some. It would have been better if `empty` was not there: It seems the feature of mapping different source and target indices is not used, so removing all `empty` lines we'd have something usable `<index> <"copy" or frequency>`. We could parse it like the proposed bob.link format where `copy` will be `-1` but that doesn't make sense for the link-format where only numbers are allowed. Could be done though, maybe via a custom header in the text file (check palette animations in libsiedler2) or a function parameter `handleCopy` (or so)

**Comment by Flow86** [comment]
yes of course we would have to extend it, and modernize it (to ignore those copy lines)

**Comment by Richard-L** [comment]
Since sampling rate is brought up, in fact all RttR SFX sound damper compared to vanilla S2 to me, as if the sampling rate was too low. Sometimes the pitch is also off, for example with the "baa" of the sheep I can hear this consistently. Happy to upload another recording if you need.

**Comment by Flow86** [comment]
the problem is, that there is no wav header in the file itself. thats why we create one, with the sample rate set from sound.scs, then we upsample it to 44100 so SDL(-mixer) can play them.

**Comment by Richard-L** [comment]
[farmer-sfx.zip](https://github.com/Return-To-The-Roots/s25client/files/4774560/farmer-sfx.zip)

^comparison of farmer sound from vanilla S2 and RttR.

Currently most SFX are either too unclear in sound quality, or play back at wrong speed.

**Comment by Spikeone** [comment]
@Richard-L @Flamefire any news on that? Thought you changed the sound.scs some time ago :)

**Comment by Flamefire** [comment]
I just changed the format a bit, not the values. Someone with experience with sounds can do that, see https://github.com/Return-To-The-Roots/s25client/issues/1246#issuecomment-638884713 :)

**Comment by Richard-L** [comment]
@Spikeone not possible to get good quality SFX like you hear on Dropbox, so I lost interest.
